feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -43,17 +43,17 @@ func TestMain(m *testing.M) {
 // deviation is set, in which case it uses check.EqualOrNil to allow the device
 // to return a nil value. This should only be used when `val` is the default
 // for this particular query.
-func EqualToDefault[T any](query ygnmi.SingletonQuery[T], val T) check.Validator {
-	if *deviations.MissingValueForDefaults {
+func EqualToDefault[T any](query ygnmi.SingletonQuery[T], val T, dut *ondatra.DUTDevice) check.Validator {
```

Comment:
Sorry that this creates more work for you, but I think these helpers would be better to not pass the full DUT in if we only need the value of a deviation. Could you change EqualToDefault and CheckPresence instead accept a boolean called missingDefaults and pass that in when called?

Reply:
done